src/shared/data/scatter-chart-data.js

```diff
@@ -1,32 +1,25 @@
 const scatterChartData = [
-  { id: 'Legend 1', x: 20, y: 0.35 },
-  { id: 'Legend 1', x: 30, y: 0.6 },
-  { id: 'Legend 1', x: 40, y: 0.275 },
-  { id: 'Legend 1', x: 50, y: 0.775 },
-  { id: 'Legend 1', x: 60, y: 0.675 },
-  { id: 'Legend 1', x: 70, y: 0.75 },
-  { id: 'Legend 1', x: 80, y: 0.450 },
-  { id: 'Mammamia', x: 20, y: 0.3 },
-  { id: 'Mammamia', x: 30, y: 0.375 },
-  { id: 'Mammamia', x: 40, y: 0.450 },
-  { id: 'Mammamia', x: 50, y: 0.650 },
-  { id: 'Mammamia', x: 60, y: 0.775 },
-  { id: 'Mammamia', x: 70, y: 0.650 },
-  { id: 'Mammamia', x: 80, y: 0.5 },
-  { id: 'Third Legend Longer', x: 20, y: 0.2 },
-  { id: 'Third Legend Longer', x: 30, y: 0.425 },
-  { id: 'Third Legend Longer', x: 40, y: 0.375 },
-  { id: 'Third Legend Longer', x: 50, y: 0.575 },
-  { id: 'Third Legend Longer', x: 60, y: 0.6 },
-  { id: 'Third Legend Longer', x: 70, y: 0.550 },
-  { id: 'Third Legend Longer', x: 80, y: 1 },
-  // { id: 'Fourth Legend', x: 20, y: 0.2 },
-  // { id: 'Fourth Legend', x: 30, y: 0.425 },
-  // { id: 'Fourth Legend', x: 40, y: 0.375 },
-  // { id: 'Fourth Legend', x: 50, y: 0.575 },
-  // { id: 'Fourth Legend', x: 60, y: 0.6 },
-  // { id: 'Fourth Legend', x: 70, y: 0.550 },
-  // { id: 'Fourth Legend', x: 80, y: 0.9 },
+  { id: 'Arts & Entertainment', x: 20, y: 0.35 },
```

Comment:
just noticed this file. all these non-implementation files (such as test data for stories) should be moved out of `src/`. otherwise these are only increasing the package size.

Reply:
I am thinking to move all data folder in stories. Is that ok?